geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/distance/CoordinateCircleDistanceFn.java

```diff
@@ -12,7 +12,7 @@
 		DistanceFn<Coordinate>
 {
 
-	protected static CoordinateReferenceSystem DEFAULT_CRS;
+	protected static final CoordinateReferenceSystem DEFAULT_CRS;
```

Comment:
now throws a runtime error in the static initializer if 4326 can't be loaded;  but really, what else can we do?  We are sort of dead in the water, so that's probably the best alternative?